h5netcdf/tests/test_h5netcdf.py

```diff
@@ -268,7 +276,7 @@ def read_legacy_netcdf(tmp_netcdf, read_module, write_module):
 
 
 def read_h5netcdf(tmp_netcdf, write_module):
-    remote_h5 = True if tmp_netcdf.startswith(('http:', 'hdf5:')) else False
+    remote_h5 = True if tmp_netcdf.startswith(REMOTE_H5_PATH_PREFIX) else False
```

Comment:
should just be: `remote_h5 = tmp_netcdf.startswith(REMOTE_H5_PATH_PREFIX)`

Reply:
Done.